src/main/java/com/direwolf20/buildinggadgets/client/gui/GuiMod.java

```diff
@@ -65,11 +65,22 @@
             return true;
         }
         return false;
-    });
+    }),
+
+    MATERIAL_LIST(player -> {
+        ItemStack mainhand = player.getHeldItemMainhand();
+        if (mainhand.getItem() instanceof ITemplate)
```

Comment:
In order to adapt to the api this needs to check for the TemplateKey capability - this is the legacy interface